x509-cert/src/serial_number.rs

```diff
@@ -0,0 +1,74 @@
+//! X.509 serial number
+
+use der::{
+    asn1::Uint, DecodeValue, EncodeValue, ErrorKind, FixedTag, Header, Length, Reader, Result, Tag,
+    ValueOrd, Writer,
+};
+
```

Comment:
Maybe make this an inherent constant?

Reply:
not too sure what you mean here.

Reply:
Do you mean move that to another module named `constant`? (`const-oid`? not really an oid)
Do you mean expose the `SERIAL_NUMBER_MAX_LEN`? like `pub const` and re-export in the `src/lib`?

Reply:
like `SerialNumber::MAX_LEN`

Reply:
oh, right. yeah makes sense.